test/Serilog.ApprovalTests/Serilog.approved.txt

```diff
@@ -120,6 +120,8 @@ namespace Serilog.Core
     }
     public sealed class Logger : Serilog.Core.ILogEventSink, Serilog.ILogger, System.IAsyncDisposable, System.IDisposable
     {
+        public System.Action<Serilog.Events.LogEvent>? AfterDispatch { get; set; }
```

Comment:
Is it possible that these changes are acceptable in the light of the v3 release? Moreover, backward compatibility is not violated here.